Microsoft.Toolkit/Microsoft.Toolkit.csproj

```diff
@@ -16,10 +16,16 @@
     <DebugType>Full</DebugType>
   </PropertyGroup>
 
-  <ItemGroup>
+  <!-- .NET Standard 2.0 doesn't have the Span<T> type -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="System.Memory" Version="4.5.3" />
   </ItemGroup>
 
+  <!-- .NET Standard 2.1 doesn't have the Unsafe type -->
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
```

Comment:
This was not being used by .Net Standard 2.0, and I don't see any references in this PR to any unsafe specific code. Is this still needed?

Reply:
It was being referenced indirectly, since it's one of the dependencies of `System.Memory`. The `Unsafe` class is being used in some helpers, like [here](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/3a9b0820361f46bf5d2c09f043ed1ec2703ecabc/Microsoft.Toolkit/Extensions/ValueTypeExtensions.cs#L58) in `ValueTypeExtensions.ToHexString` and [here](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/ddf8ee05422eb9f45503a7e2f74a2ee51bbfbfff/Microsoft.Toolkit/Diagnostics/Guard.Comparable.Generic.cs#L110) in `Guard.IsBitwiseEqualTo` 😄

Reply:
But those classes are not NS2.1 only, so NS2.0 also uses it. So NS2.0 has the unsafe and NS2.1 doesn't?

Reply:
.NET Standard 2.0 is getting the `Unsafe` class as an indirect dependency from the `System.Memory` package, while .NET Standard 2.1 needs to reference the `Unsafe` package directly because it doesn't reference `System.Memory`, as those APIs (eg. `Span<T>`) come right out of the box on .NET Standard 2.1. Maybe I should add a more detailed comment in the `.csproj` file explaining this, do you think that would be useful?

Reply:
I think it is ok the way it is. It's building, and if you remove it it won't, so it is clear hehe. Thanks!

Reply:
Ahahahah awesome, no problem! 😄